src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The bulldozer stopped working");

```

Comment:
Similar to the `doWork` method, the message here should also be more informative. Include the class name to indicate that it's a Bulldozer that stopped working.